<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Attendance2_NewEntry</class>
 <widget class="QWizard" name="Attendance2_NewEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employee Attendance</string>
  </property>
  <property name="options">
   <set>QWizard::CancelButtonOnLeft|QWizard::DisabledBackButtonOnLastPage|QWizard::HelpButtonOnRight</set>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <property name="title">
    <string>Employee Attendance</string>
   </property>
   <property name="subTitle">
    <string>To start the attendance data entry, please enter the shift and shift dates below.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <widget class="QTimeEdit" name="dtpIntroTimeIn">
      <property name="displayFormat">
       <string>hh:mm</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Date Out:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QDateEdit" name="dtpDateIn">
      <property name="displayFormat">
       <string>dd MMM yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="cboShift"/>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Notes:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QDateEdit" name="dtpDateOut">
      <property name="displayFormat">
       <string>dd MMM yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Shift:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Date In:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="QTextBrowser" name="txtNotes"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Time In:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Time Out:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QTimeEdit" name="dtpIntroTimeOut">
      <property name="displayFormat">
       <string>hh:mm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage">
   <property name="title">
    <string>Employees</string>
   </property>
   <property name="subTitle">
    <string>Select the employees in this list.</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="cmdReloadEmployees">
        <property name="text">
         <string>Reload</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="trvEmployees">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Roll No.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage_2">
   <property name="title">
    <string>Attendance Entry</string>
   </property>
   <property name="subTitle">
    <string>Enter the clock times for the selected employees...</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableWidget" name="tblTimes">
      <column>
       <property name="text">
        <string>Roll No</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time In</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time Out</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>EmployeeID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Comment</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage_3">
   <property name="title">
    <string>Confirm Details</string>
   </property>
   <property name="subTitle">
    <string>Confirm that the details below are OK, then click on 'Save'</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTextBrowser" name="txtReview"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="cmdSave">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset resource="rc.qrc">
          <normaloff>:/images/FloppyDisk.png</normaloff>:/images/FloppyDisk.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <property name="title">
    <string>Complete</string>
   </property>
   <property name="subTitle">
    <string>The overtime entries have been saved. Please click on Exit to close this page, or Start Over to enter a new set.</string>
   </property>
   <widget class="QPushButton" name="cmdExit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdStartOver">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Over</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
